ami_cleanup.py

```diff
@@ -18,25 +27,11 @@
 
 ec = boto3.client('ec2', os.environ['region'])
 ec2 = boto3.resource('ec2', os.environ['region'])
-images = ec2.images.filter(Owners=[os.environ['ami_owner']])
-
-def lambda_handler(event, context):
-
-    reservations = ec.describe_instances(
-        Filters=[
-            {'Name': 'tag-key', 'Values': ['backup', 'Backup', 'Snapshot']},
-        ]
-    ).get(
-        'Reservations', []
-    )
+images = ec2.images.filter(Owners=[os.environ['ami_owner']],  Filters=[{'Name': 'tag-key', 'Values': ['AMIDeleteOn']}])
```

Comment:
This does not filter on instance id. Won't there be multiple lambdas deployed at the same time? How do we keep them from stepping on each other? @comeanother